.github/workflows/gitlab.yml

```diff
@@ -3,22 +3,30 @@ on:
   pull_request_target:
     types: [labeled]
   push:
+  schedule:
+    # Avoid running on 0 since a lot of other workflows on github start at that
+    # time and this can cause delays or even dropping of jobs:
+    # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule
+    - cron: 18,48 18-23 * * *
+    - cron: 17,47 0-5 * * *
 permissions: {}
 
 jobs:
   trigger-gitlab-pipeline:
     runs-on: [self-hosted, libtelio]
     if: |
+      github.event.schedule ||
```

Comment:
can you please use: `github.event_name == 'schedule'` to be consistent with the rest of the `if`? Same in the second place. Sometimes different events can have the same payload type so this is clearer IMO

Reply:
Good point! Done :)